libqnotero/_themes/default.py

```diff
@@ -41,7 +41,7 @@ def __init__(self, qnotero):
 		self.setWindowProperties()
 		self.setScrollBars()
 		
-	def icon(self, iconName):
+	def icon(self, iconName, overrideIconExt=None):
```

Comment:
Other themes should be updated as well. Right now this breaks the chameleon theme.